aiohttp/helpers.py

```diff
@@ -42,6 +43,39 @@ def __new__(cls, login, password='', encoding='latin1'):
 
         return super().__new__(cls, login, password, encoding)
 
+    @classmethod
+    def decode(cls, auth_header, encoding='latin1'):
+        """Create a :class:`BasicAuth` object from an ``Authorization`` HTTP
+        header.
+
+        :param auth_header:  The value of the ``Authorization`` header.
+        :type auth_header:  str
```

Comment:
You can user `:param str auth_header: ...` today, but aiohttp doesn't uses docstring in code, but separate files under doc/ directory.